services/template-generator/.env.example

```
@@ -0,0 +1,80 @@
+# Template Generator Service Configuration
+
+# Server Configuration
+PORT=3000
+HOST=0.0.0.0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)